rundeckapp/src/groovy/com/dtolabs/rundeck/app/internal/logging/FSStreamingLogReader.groovy

```diff
@@ -45,10 +48,21 @@ class FSStreamingLogReader implements ReverseSeekingStreamingLogReader {
         this.rundeckLogFormat=rundeckLogFormat
     }
     private void detectLegacyLogFile(){
-        file.withReader('UTF-8') {reader->
+
+        if(file.getName().contains(LoggingService.LOG_FILE_FILETYPE)) {
```

Comment:
I think this should check for `file.getName().endsWith('.'+LoggingService. LOG_FILE_FILETYPE_UNCOMPRESSED)`